app/src/androidTest/java/org/worldbank/transport/driver/DatastoreTests/RecordTests.java

```diff
@@ -28,6 +28,8 @@ protected void setUp() throws Exception {
 
         // create app in test context
         app = new DriverApp(true);
+        // TODO: figure out how to set context to get asset directory to be able to re-enable these tests
```

Comment:
I'd move this up to the same place as where the comment starts so that it's clear why this is commented out.